stl/inc/xutility

```diff
@@ -848,9 +848,23 @@ _INLINE_VAR constexpr bool _Is_input_iter_v = is_convertible_v<_Iter_cat_t<_Iter
 template <class _Iter>
 _INLINE_VAR constexpr bool _Is_fwd_iter_v = is_convertible_v<_Iter_cat_t<_Iter>, forward_iterator_tag>;
 
+template <class _Iter>
+_INLINE_VAR constexpr bool _Is_ranges_fwd_iter_v =
+#if defined(__cpp_lib_concepts)

```

Comment:
We conventionally `#ifdef MEOW`/`#ifndef MEOW` instead of `#if defined(MEOW)`/`#if !defined(MEOW)`.  (Also on 863.)